packages/perseus/src/widgets/radio/choice.jsx

```diff
@@ -203,218 +130,240 @@ class Choice extends React.Component<$FlowFixMe, State> {
     //
     // This enables us to use shorthand inside this component, while
     // maintaining a consistent API for the parent.
-    _sendChange: ({|checked?: boolean, crossedOut?: boolean|}) => void = (
-        newValues,
-    ) => {
-        const checked = newValues.checked ?? this.props.checked;
-        const crossedOut = newValues.crossedOut ?? this.props.crossedOut;
-        this.props.onChange({checked, crossedOut});
-    };
-
-    /**
-     * Public method. Focus the choice's <input> element.
-     */
-    focusInput: () => void = () => {
-        this._input?.focus();
-    };
-
-    render(): React.Node {
-        const sat = this.props.apiOptions.satStyling;
-
-        const {reviewMode, correct, checked} = this.props;
-        // HACK: while most of the styling for rendering SAT items is handled
-        // via aphrodite, we also need to assign normal CSS classnames here to
-        // special-case the coloring of MathJax formulas (see .MathJax .math in
-        // stylesheets/task-package/tasks.less)
-        const satCorrectChoice = sat && reviewMode && correct;
-        const satIncorrectChecked = sat && reviewMode && !correct && checked;
-        const descriptionClassName = classNames(
-            "description",
-            satCorrectChoice && "sat-correct",
-            satIncorrectChecked && "sat-incorrect",
-            css(
-                !sat && styles.description,
-                sat && styles.satDescription,
-                satCorrectChoice && styles.satDescriptionCorrect,
-                satCorrectChoice &&
-                    checked &&
-                    styles.satDescriptionCorrectChecked,
-                satIncorrectChecked && styles.satDescriptionIncorrectChecked,
-            ),
-        );
-
-        const rationaleClassName = classNames(
-            "perseus-radio-rationale-content",
-            css(
-                styles.rationale,
-                !sat && styles.nonSatRationale,
-                sat && styles.satReviewRationale,
-            ),
-        );
-
-        // We want to show the choices as dimmed out when the choices are
-        // disabled. However, we don't want to do this in the SAT product and
-        // we also don't want to do this when we're in review mode in the
-        // content library.
-        const showDimmed =
-            (!sat && !reviewMode && this.props.apiOptions.readOnly) ||
-            this.props.crossedOut;
-
-        return (
+    function sendChange(newValues: {|
+        checked?: boolean,
+        crossedOut?: boolean,
+    |}) {
```

Comment:
Any particular reason? I know they're subtly different and in-vogue, but I consider arrow functions more difficult to parse (especially with Flow). I don't mind changing it, just curious.

Reply:
Using arrow functions avoids issues involving `this`.  `this` isn't used here so it isn't necessary here, but it would be more consistent.  I get what you're saying about arrow functions being hard to parse when types are involved.  I updated my initial comment to be non-blocking so don't feel that you need to make this change.